app/src/main/java/com/example/tmdb/modules/FavoriteMoviesBase.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.tmdb.modules
+
+class FavoriteMoviesBase {
```

Comment:
This should be hidden behind an interface. That way, other classes will depend on that interface abstraction, not the concrete implementation.